build.sbt

```diff
@@ -87,7 +87,7 @@ lazy val publishSettings : Seq[Setting[_]] = Seq(
 // should not be set directly. It is the same as the Maven version and derived automatically from `baseVersion` and
 // `baseVersionSuffix`.
 globalVersionSettings
-baseVersion in Global := "2.12.1"
+baseVersion in Global := "2.13.0-M1"
```

Comment:
Here's the problem. `baseVersion` needs to be `2.13.0`. `M` or `RC` suffixes are not part of SHA-versioned snapshots. An actual M1 release will get `M1` as the `baseVersionSuffix`. See https://github.com/scala/scala/blob/2.12.x/project/VersionUtil.scala#L49-L63 for the allowed combinations of values.